server/src/server.rs

```diff
@@ -332,7 +321,7 @@ impl Drop for ServerStream {
 }
 
 struct CubebServerCallbacks {
```

Comment:
This is unrelated to the changes here, but could we rename this to `DeviceCollectionChangeCallback` or `DeviceCallback`, or something similar? `callback` appears in lots of the classes or variables in this crate, and each of them has a different meaning. I find it hard to read know what this is without jumping into the definitions. This bothers me when reading the code here.

Reply:
That's clearer, I've renamed it.